packages/autorest.python/tasks.py

```diff
@@ -129,7 +129,7 @@ class Config(NamedTuple):
         "namespace": "vanilla.body.array"
     },
     "BodyByteWithPackageName": {
-        "package-name": "package-name",
+        "package-name": "packagename",
```

Comment:
According to [guideline of Azure SDK distribution packages](https://azure.github.io/azure-sdk/python_design.html#azure-sdk-distribution-packages), package name shall keep consistent with namespace.